native/cocos/renderer/pipeline/custom/NativeResourceGraph.cpp

```diff
@@ -215,9 +215,12 @@ void ResourceGraph::mount(gfx::Device* device, vertex_descriptor vertID) {
             }
             CC_ENSURES(buffer);
         },
-        [&](const IntrusivePtr<gfx::Texture>& texture) {
-            CC_EXPECTS(texture);
-            std::ignore = texture;
+        [&](IntrusivePtr<gfx::Texture>& texture) {
+            if (!texture) {
+                auto info = getTextureInfo(desc);
+                texture = device->createTexture(info);
+            }
+            CC_ENSURES(texture);
```

Comment:
When will this texture be null?

Reply:
I dont know, you added these codes for me ?

Reply:
We might revert this change and run the code. I believe it will not crash.

Reply:
ok

Reply:
These codes cannot be removed. Will be refactored in 3.9.